vanilla-ge/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7.7-slim
+
+RUN apt-get update
+
+ADD ./requirements.txt /app/
+RUN pip install -r /app/requirements.txt
+WORKDIR /app
+
+LABEL maintainer="BERA"
+
+# Run great_expectations by default and allow any command to be run if desired.
+CMD ["great_expectations"]
```

Comment:
I think this should be the entrypoint for the vanilla-ge

Reply:
Agreed. Though sometimes I get lazy and just want to invoke a command in a container without the `--entrypoint` arg. 😄